Contents/mods/Hydrocraft/media/lua/server/RockHCExtra.lua

```diff
@@ -21,48 +21,64 @@ skill = player:getPerkLevel(Perks.PlantScavenging);
     if chance <= dirtLvl10 then
         secondChance = ZombRand(2);
         if secondChance == 0 then player:getInventory():AddItem("Hydrocraft.HCSandstone");
-        elseif secondChance == 1 then player:getInventory():AddItem("Hydrocraft.HCTrashlarge");	
+            player:getInventory():AddItems("Hydrocraft.HCRedclay",1);
```

Comment:
not sure about the changes to add clay, you are meant to get the clay from the quarry